tests/pages/services/TasksTable-cy.js

```diff
@@ -87,4 +87,47 @@ describe("Tasks Table", function() {
       });
     });
   });
+
+  context("Service tasks checkbox", function() {
+    beforeEach(function() {
+      cy.configureCluster({
+        mesos: "1-service-with-executor-task"
+      });
+      cy.visitUrl({ url: "/services/detail/%2Fcassandra/tasks?_k=rh67gf" });
+      cy.get("table tr").find(".form-element-checkbox").as("checkboxes");
+    });
+
+    function assertCheckboxLength() {
+      cy.get("@checkboxes").should("have.length", 2);
+    }
+
+    it("Select all tasks available", function() {
```

Comment:
These tests won't protect against the issue of checkboxes loosing their checked state. In each of these tests, could add a delay with `cy.wait(5000)` and make sure the checkbox is still checked? Would protect against regression.